packages/core/lottie-react-native.podspec

```diff
@@ -19,24 +19,26 @@ Pod::Spec.new do |s|
 
   s.source                  = { :git => "https://github.com/lottie-react-native/lottie-react-native.git", :tag => "v#{s.version}" }
   s.source_files            = "ios/**/*.{h,m,mm,swift}"
-  s.exclude_files           = "ios/Fabric"
 
-  s.dependency 'React-Core'
   s.dependency 'lottie-ios', '~> 3.5.0'
 
-  s.compiler_flags  = folly_compiler_flags
-
-  s.pod_target_xcconfig    = {
-    "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
-    "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
-  }
-
   if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
-    s.subspec "fabric" do |ss|
-      ss.dependency "React-RCTFabric"
-      ss.dependency "React-Codegen"
-      ss.source_files       = "ios/Fabric/**/*.{h,m,mm,swift}"
-      ss.exclude_files      = "ios/**/*.{swift}"
-    end
```

Comment:
I am wondering why you removed that part. I actually added that so the fabric files would be provided via a subspec and would have a separate entry in the podfile.lock.

Reply:
Hey, I can bring it back. I preferred to have this as similar as possible to the one Meta has been recommending. It also keeps it consistent with other libraries.

Reply:
Let's keep it as recommended and it would help ensure that the library is consistent with other libraries and that any potential bugs are more likely to be discovered and fixed more quickly.